### PR TITLE
feat(Skeleton): add and expose raduis properties

### PR DESCRIPTION
**Pull request by mshaaban0**
# Purpose of PR

* Adds default border radius of `4px` to Skeleton components
* Exposes `radiusX` and `radiusY` property to Skeleton components

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/main/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

**Comment by denkristoffer** [comment]
I think I prefer the skeletons for text without the radius, but I'll let @fabe decide 😄

File: packages/components/skeleton/src/SkeletonText/SkeletonText.tsx [hunk 1]
```diff
@@ -23,6 +24,14 @@ export interface SkeletonTextProps {
...
    */
   marginBottom?: stringOrNumber;
+  /**
+   * X-axis border radius (in number)
+   */
+  radiusX?: stringOrNumber;
+  /**
+   * Y-axis border radius (in number)
+   */
+  radiusY?: stringOrNumber;
```
Comment (bgutsol) [inline packages/components/skeleton/src/SkeletonText/SkeletonText.tsx:34]: is there a use case for a different border-radius between X and Y. Can we just expose one property that will be used for both values?

Comment (bgutsol) [reply packages/components/skeleton/src/SkeletonText/SkeletonText.tsx:34]: ah, I see that we already had it as a separate props in the SkeletonImage

**Review by bgutsol — APPROVED** [review]

**Comment by fabe** [comment]
> I think I prefer the skeletons for text without the radius, but I'll let @fabe decide 😄

With v4 we added border radius to pretty much every block element, I think we just forgot to do it in this case. But we can always reevaluate this later and see if it works better or not!

**Review by fabe — APPROVED** [review]

**Pull request merged**